src/core/libmaven/PeakDetector.cpp

```diff
@@ -350,9 +344,7 @@ void PeakDetector::processSlices(vector<mzSlice*> &slices, string setName)
             break;
 
         mzSlice* slice = slices[s];
-        Compound* compound = slice->compound;
-        if (compound != nullptr && compound->hasGroup())
-            compound->unlinkGroup();
```

Comment:
Because you said now we don't link compounds with groups and you told me to remove these functions and corresponding code where those functions were being used.

Reply:
Yes. Sorry, I only saw this later and forgot to delete this comment.